scanner/src/main/kotlin/scanners/FileCounter.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner.scanners
+
+import com.here.ort.model.Provenance
+import com.here.ort.model.ScanResult
+import com.here.ort.model.ScanSummary
+import com.here.ort.model.ScannerDetails
+import com.here.ort.model.jsonMapper
+import com.here.ort.scanner.LocalScanner
+
+import java.io.File
+import java.time.Instant
+
+/**
+ * A simple [LocalScanner] that only counts the files in the scan path. It is useful for testing because it is much
+ * faster than the other scanners.
+ */
+object FileCounter : LocalScanner() {
+    data class FileCount(val fileCount: Int)
```

Comment:
Is it worth having that class (IMO `fileCount.fileCount` reads odd below), or can we just re-use the `Scanner.Result` data class with unused properties set to dummy values?

Reply:
I think having this class is cleaner than having a dependency on another class intended for another purpose. I did some renaming, maybe you like that better.